draft-ietf-rtcweb-jsep.xml

```diff
@@ -2963,7 +2867,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
           <t>Next, the SessionDescription is parsed into a data
           structure, as described in the
-          <xref target="sec.parsing-a-desc" /> section below. If parsing
+          <xref target="sec.parsing-a-desc" /> section below.
```

Comment:
This is the local description section. I think this would probably be most appropriate at the end of 5.7.1 and 5.7.2 where it talks about parsing.